src/main/java/step3/data/Car.java

```diff
@@ -0,0 +1,18 @@
+package step3.data;
+
+public class Car {
+
+  public Integer position;
+
+  public Car() {
+    this.position = 0;
+  }
+
+  public boolean move(int randomNum) {
+    if (randomNum >=4) {
+      this.position ++;
+      return true;
+    }
+    return false;
+  }
```

Comment:
```suggestion
  public boolean move(MoveStrategy moveStrategy) {
    if (moveStrategy.isMove()) {
      position += 1;
    }
  }
```
이렇게 사용될 수 있도록 전략패턴으로 구성해보면 어떨까요?

Reply:
전략패턴이라는 것에 익숙치 않아 나름 조사후 적용해보았습니다.
리뷰에서 의도하신대로 했는지 잘 모르겠습니다...